doc-Installing_on_AWS/topics/installation.adoc

```diff
@@ -181,17 +181,21 @@ To import the appliance:
 ]
 ----
 +
-. Use AWS-CLI tools to import the image. Run the following command:
+. Use AWS-CLI tools to import the disk as a snapshot by following the steps in the
+link://docs.aws.amazon.com/vm-import/latest/userguide/vmimport-import-snapshot.html[Importing a Disk as a Snapshot Using VM Import/Export] documentation.
 +
 ----
-$ aws ec2 import-image --disk-containers file://containers.json --region BUCKET_REGION
+$ aws ec2 import-snapshot
 ----
 +
-. Check the progress of your imported image by running the following command:
+. Create the AMI from the snapshot by following the steps in the
+link://docs.aws.amazon.com/AWSEC2/latest/UserGuide/creating-an-ami-ebs.html[Creating a Linux AMI from a Snapshot] documentation.
 +
 ----
  $ aws ec2 describe-import-image-tasks --region BUCKET_REGION --import-task-ids import-ami-AMI_ID_GOT_FROM_RESPONSE
```

Comment:
This needs to be changed to either steps to do in UI (that's how I tested), different aws ec2 command (or just a link to aws doc) as `describe-import-image-tasks` is a wrong command...

Reply:
Will you please indicate the correct command to create an AMI? I'd like to have that example command in here for consistency with the other steps -- rather than only reference them to the AWS documentation.

Reply:
> Will you please indicate the correct command to create an AMI?

The doc linked talks about using `aws ec2 register-image` command. You can see all the available options in the doc: https://docs.aws.amazon.com/cli/latest/reference/ec2/register-image.html

Since I always use Console to do this step, I don't know which options are really required when creating AMI from snapshot though...